src/layouts/split/types.ts

```diff
@@ -0,0 +1,14 @@
+
+type ChildType = React.ReactElement | Element
+
+type GridStyleType = {
+  backgroundColor: string,
+  color: string
+}[]
+
+export type SplitLayoutTypes = {
+  children: [ChildType, ChildType] | [ChildType],
+  columns?: number,
+  gridStyles?: GridStyleType,
+  grid?: [number] | [number, number],
+}
```

Comment:
we can get rid of `ChildType`

And make `children: React.ReactNode`, which encapsulates `React.ReactElement`. It does necessarily include `Element` but `Element` is too broad for our use case anyway (its almost like importing `any`)

Reply:
you should also include `import {reactNode} from 'react'